games.yaml

```diff
@@ -1670,8 +1797,19 @@
       info: moderately active development, C++/Lua
       added: 2013-06-08
 
+- name: "Stunt Car Racer"
+  clones:
+    - url: http://sourceforge.net/projects/stuntcarremake/
+      repo: git://git.code.sf.net/p/stuntcarremake/code
+      name: Stunt Car Racer Remake
+      added: 2015-04-04
+      info: stalled development, C++
+      media:
+        - url: https://a.fsdn.com/con/app/proj/stuntcarremake/screenshots/SCR2.png
 
-- name: [Stunts, Stunts (video game)]
+- name: 
+  - [Stunts, Stunts (video game)]
+  - [4D Sports Driving]
```

Comment:
This shouldn't be wrapped in brackets, since it's only one string and template fails there (`wiki` becomes `None`).